src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Application.cshtml

```diff
@@ -37,7 +41,7 @@
 
         <dl class="govuk-body app-dl-icon-list govuk-!-margin-bottom-4">
             <dt class="app-dl-icon-list__title">
-                @*@if (Model.IsLocationMatch) { <span class="app-dl-icon-list__icon app-dl-icon-list__icon--tick">Match for: </span> }*@
+                @if (Model.IsLocationMatch) { <span class="app-dl-icon-list__icon app-dl-icon-list__icon--tick">Match for: </span> }
```

Comment:
I assume this just came from whatever you branched off

Reply:
yeah I only added the button to this view so it will have been a merge issue

Reply:
that I cocked up at a guess

Reply:
I've compared with the previous one that Conor did and made it the same, I think I added stuff in that shouldn't have when I merged, so this is resolved